sdk/containerregistry/service.projects

```diff
@@ -7,12 +7,16 @@
   the set of projects to match what type of SDK we are interested in building.
 
   SDKType values
-   - empty or "client" will build the new track 2 client/data libraries
+   - "all" or "client" will build the new track 2 client/data libraries
    - "data" will build the track 1 data plane libraries
    - "mgmt" will build the track 1 mgmt libraries
 -->
 
 <Project>
+  <ItemGroup Condition="'$(SDKType)' == 'all'">
+    <ProjectsToInclude Include="$(MSBuildThisFileDirectory)**\*.csproj" />
+  </ItemGroup>
+
```

Comment:
Should we remove the empty clause on line 20, since it should now not be possible to hit?

Reply:
Good point.